packages/runtime-dom/types/globalDirectives.d.ts

```diff
@@ -0,0 +1,10 @@
+// Note: this file is auto concatenated to the end of the bundled d.ts during
```

Comment:
`globalComponents.d.ts`, `globalDirectives.d.ts` files name seem not match to content.

Reply:
What do you recommend? just `components` and `directives` or other?

Reply:
Sorry i didn't make it clear, I mean `globalComponents.d.ts` file content is `interface GlobalDirectives`, `globalDirectives.d.ts` file content is `interface GlobalComponents`.